examples/step-20/doc/results.dox

```diff
@@ -265,8 +265,8 @@ KInverse<dim>::value_list (const std::vector<Point<dim> > &points,
 
       double permeability = 0;
       for (unsigned int i=0; i<centers.size(); ++i)
-        permeability += std::exp(-(points[p]-centers[i]).square()
-                                 / (0.1 * 0.1));
```

Comment:
Thanks for spotting this! I guess you are missing some braces, though. Currently, updating `permeability` is not inside the body of the for-loop.
Alternatively, you could probably also use `Tensor::norm_square()`.

Reply:
### Thank you! Sorry for the missing braces - bad merge on my part, thank you for catching it!